recipe-server/client/control/components/MultiPicker.js

```diff
@@ -0,0 +1,234 @@
+import React, { PropTypes as pt } from 'react';
+
+export default class MultiPicker extends React.Component {
```

Comment:
Yeah, I'm going to need your help with implementing FieldArrays, @Osmose . I've been attempting to implement it for the [CheckboxGroup component](https://gist.github.com/andymikulski/5a4497f97354f8340761453a8848cd1a), however I'm running into several issues:

- `fields` never has any getters, despite the [documentation](http://redux-form.com/6.5.0/docs/api/FieldArray.md/) indicating there should be `getAll`, `get`, etc
- Pushing values to `fields` is ignored. It's weird - if I do `fields.push('whatever')`, it simply sets `fields` to `channels[0]` (where `channels` is the field name). Pushing again adds `channels[1], channels[2], ...` and so on - always ignoring the actual values.

I have a feeling I'm just getting the whole thing wrong; there seems to be increased complexity with this approach, and things like the getters missing is a big red flag for me. Any direction you could give would be appreciated.